Containerfile

```diff
@@ -113,16 +149,14 @@ CMD ["bundle", "exec", "rails", "server", "-b", "0.0.0.0", "-p", "3000"]
 ## Development
 FROM builder AS development
 
+USER "${APP_USER}"
+
 COPY --chown="${APP_USER}":"${APP_USER}" ./spec "${APP_HOME}"/spec
 COPY --from=builder /usr/local/bin/dockerize /usr/local/bin/dockerize
 
-RUN chown "${APP_USER}":"${APP_USER}" -R "${APP_HOME}"
-
-USER "${APP_USER}"
-
 RUN bundle config --local build.sassc --disable-march-tune-native && \
     bundle config --delete without && \
-    bundle install --deployment --jobs 4 --retry 5 && \
+    BUNDLE_FROZEN=true bundle install --deployment --jobs 4 --retry 5 && \
```

Comment:
not applicable on this PR but for development, wouldn't we do the install without `--deployment` option?